SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -47,55 +44,55 @@ public class AppRemotingSample : MonoBehaviour
         [SerializeField, Tooltip("The configuration information for listening to remote connection.")]
         private Remoting.RemotingListenConfiguration remotingListenConfiguration = new Remoting.RemotingListenConfiguration { ListenInterface = "0.0.0.0", HandshakeListenPort = 8265, TransportListenPort = 8266, MaxBitrateKbps = 20000 };
 
-        private static readonly List<XRDisplaySubsystem> XRDisplaySubsystems = new List<XRDisplaySubsystem>();
-        private static bool s_connected = false;
-        private static DisconnectReason s_disconnectReason = DisconnectReason.None;
+        private bool m_connected = false;
+        private DisconnectReason m_disconnectReason = DisconnectReason.None;
         private bool m_remotingInProgress = false;
         private AppRemotingMode m_appRemotingMode = AppRemotingMode.none;
-        private bool m_showFlatUI = true;
+        private bool m_expandUIByDefault = true;
 
         private void Awake()
         {
             // The app remoting flat UI menu is collapsed in editor. User can click on it to expand the menu and enter the details for app remoting.
             // The purpose of collapsible UI is for MRTK in-editor simulation.
             if (Application.isEditor)
             {
-                m_showFlatUI = false;
+                m_expandUIByDefault = false;
             }
 
```

Comment:
at this point, the "m_connected" is still the default value "false" in above member definition.  Don't we need to query for the latest property here?  What if the app get connected in a separate page (or maybe playmode remoting), and this scene is initialized after the connection.  Isn't the "m_connected" wrong?

Reply:
I was working under the assumption that this gameobject would setup _always_ before the connection starts. 

You bring up a fair point - I'll add some code here to set the status of m_connected in this Awake() method.